.github/ISSUE_TEMPLATE.md

```diff
@@ -1,4 +1,4 @@
-<!-- BUGGY OR UGLY? Please use this template.
+<!-- For ugly or incorrect code issues: Please use this template.
 
```

Comment:
How about something like

> If your issue isn’t about ugly or incorrect code, feel free to delete this and write whatever you want here.

between lines 1 and 3?

Reply:
Might be good. Might also be too long. Not sure.

Reply:
Too long.. people tent to ignore long texts.